src/jit/codegenxarch.cpp

```diff
@@ -5280,56 +5280,56 @@ void CodeGen::genConsumePutStructArgStk(GenTreePutArgStk* putArgNode, regNumber
     // The putArgNode children are always contained. We should not consume any registers.
     assert(putArgNode->gtGetOp1()->isContained());
 
-    GenTree* dst = putArgNode;
+    GenTree* dstAddr = putArgNode;
 
```

Comment:
in the function header, this line needs editing: "baseVarNum - the base for var numfor placing the "by-value" args on the stack."